### PR TITLE
add menu model

### LINKED ISSUE #25: Add model menu
We need a model for menu, to schedule meals of the year.

**It concerns only the main dish. Side dishes and salads are not included.**

## Fields

 - Date
 - Ingredients (of the main dish)

### PR DESCRIPTION
**Pull request by duckish**
fix #25

### CODE REVIEW

**Comment by maximest-pierre** [comment]
@duckish can you also do a makemigration and a migrate for the build to pass.

**Comment by RignonNoel** [comment]
Yeah, but he have problem with makemigrations because ```'menu.Ingredient',``` must be ```'meal.Ingredient',```

He work on actually

**Comment by RignonNoel** [comment]
No, you need to amend your commit. No fix commit in the project!

File: django/santropolFeast/meal/models.py [hunk 1]
```diff
@@ -67,3 +67,25 @@ class Meta:
...
+    name = models.CharField(
+        max_length=50,
+        verbose_name=_('name')
+    )
+
+    date = models.DateField(verbose_name=_('date'))
+
+    ingredients = models.ManyToManyField(
+        'menu.Ingredient',
+        related_name='related_menus'
```
Comment (manumilou) [inline django/santropolFeast/meal/models.py]: The value of the related_name property seems odd.  _menu_ would be simpler ?

Comment (RignonNoel) [inline django/santropolFeast/meal/models.py]: I'm ok with you. But some related_name respect this pattern.
I propose to open a new issue for this.

Comment (RignonNoel) [inline django/santropolFeast/meal/models.py]: I've open this issue : #96

**Comment by RignonNoel** [comment]
Thanks @duckish

**Pull request merged**